tests/test_recording/test_session_recorder.py

```
@@ -0,0 +1,100 @@
+
+        # Verify recording file was created
+        assert exit_code == 0
+        assert recorder.recording_path.exists()
+
+        # Verify file has content (header at minimum)
+        content = recorder.recording_path.read_text()
+        assert len(content) > 0
+        assert "version" in content  # Asciicast header
+
+
+def test_recording_failure_fallback(clear_recording_env):
```

Ruff (0.14.10)

Unused function argument: `clear_recording_env`

(ARG001)